dist/api_data.py

```diff
@@ -932,6 +932,9 @@ def __ge__(self, other):
     Config('prealloc', 'true', r'''
         pre-allocate log files''',
         type='boolean'),
+    Config('log_prealloc_init_count', '1', r'''
```

Comment:
Remove the `log_` prefix here. Make it similar to `prealloc` above. I'm not sure why you changed this, as it was correct before.

Reply:
I'm sorry, I forgot why I changed here. Maybe I shorted out. Haha.